src/js/components/RangeInput/StyledRangeInput.js

```diff
@@ -1,14 +1,75 @@
+import { rgba } from 'polished';
 import styled, { css } from 'styled-components';
 
 import { focusStyle, normalizeColor, parseMetricToNum } from '../../utils';
 import { defaultProps } from '../../default-props';
 
+// opacity of the bound trumps the track opacity
+const getBoundOpacity = (props, bound) => {
+  return props.theme.rangeInput &&
+    props.theme.rangeInput.track &&
+    props.theme.rangeInput.track[bound] &&
+    props.theme.rangeInput.track[bound].opacity
+    ? props.theme.rangeInput.track[bound].opacity
+    : 1;
+};
+
+const getBoundColor = (props, bound) => {
+  if (
+    props.theme.rangeInput &&
+    props.theme.rangeInput.track &&
+    props.theme.rangeInput.track[bound] &&
+    props.theme.rangeInput.track[bound].color
+  ) {
+    return rgba(
```

Comment:
Why not pass the opacity into `normalizeColor()` and let it compose the `rgba()`?

Reply:
I haven't seen how `normalizedColor` can control the opacity, it doesn't look like it is using `rgba`.

Reply:
Oops, you are correct. My mistake.